### PR TITLE
Add loop stage server to allowed origins on dev (bug 1127841)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1127841

### CODE REVIEW

File: mkt/commonplace/views.py [hunk 1]
```diff
@@ -58,10 +58,11 @@ def get_allowed_origins(request, include_loop=True):
             'https://hello.firefox.com',
             'https://call.firefox.com',
         ])
-        # On dev, include loop dev origin as well.
+        # On dev, include loop dev & stage origin as well.
         if development_server:
             allowed.extend([
                 'https://loop-webapp-dev.stage.mozaws.net',
+                'https://call.stage.mozaws.net/',
```
Comment (eviljeff) [inline mkt/commonplace/views.py]: trailing slash again :)

Comment (diox) [inline mkt/commonplace/views.py]: Damn. That's it, I'm adding a test to make sure none of the origins end with slashes.

Comment (diox) [inline mkt/commonplace/views.py]: Done.

**Comment by robhudson** [comment]
r+
